lib/http_clients/neurotech.ex

```diff
@@ -116,6 +116,14 @@ defmodule HttpClients.Neurotech do
     end
   end
 
+  defp get_score(bacen_analysis) do
+    score = get_value(bacen_analysis, "CALC_BCREDISCORE_SCORE")
+
+    if is_binary(score),
+      do: String.to_integer(score),
+      else: nil
```

Comment:
```suggestion
    if is_binary(score), do: String.to_integer(score), else: nil
```
se não rolar, acho melhor deixar o if/else normal mesmo